alert/models.py

```diff
@@ -90,14 +114,16 @@ def resub_url(self):
         return '{}/s/{}'.format(settings.PCA_URL, url.short_id)
 
     def alert(self, forced=False, sent_by=''):
-        if forced or not self.notification_sent:
+        if forced or not (self.notification_sent or self.muted or self.deleted):
             text_result = Text(self).send_alert()
             email_result = Email(self).send_alert()
             logging.debug('NOTIFICATION SENT FOR ' + self.__str__())
             self.notification_sent = True
             self.notification_sent_at = timezone.now()
             self.notification_sent_by = sent_by
             self.save()
+            if not self.auto_mute:
```

Comment:
I guess the effect on the user is the same, but isn't resubscribing better for keeping accurate data?  Using the same object for multiple alerts would overwrite the notification_sent_at field.  Also, right now is_active assumes notification_sent is false.  So I just assumed it would fit better with how it is right now if auto_mute being False results in the user being automatically resubscribed... but muted isn't set here so I changed it to this:

![image](https://user-images.githubusercontent.com/12241150/71494812-17b0a480-2818-11ea-8d80-d0c46a0d7fdc.png)

Should I do this differently?

Reply:
Like I said on Slack, you're right. We should keep it so that one Registration object will only ever trigger one notification. What you changed it to with the `self.muted` setting makes sense!